spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -106,8 +106,7 @@ public void login() {
 Event fired with the `fireAsync()` method is fired asynchronously.
 All resolved synchronous observers are called in the same thread in which `fireAsync()` was called, and all the resolved asynchronous observers are called in one or more different threads.
```

Comment:
I think this should be changed as well. No SYNC observers are called on "fireAsync()"